### ISSUE TITLE
Unable to install package due to bad version on hot-shots dependency

### ISSUE THREAD

**Issue by afittz** [DataDog/datadog-lambda-js#110]
The issue is an invalid version in the packages.json file:

hot-shots@**true**7.7.1 

should be 

hot-shots@7.7.1

## Expected Behavior
npm install should install the package

## Actual Behavior
npm install fails to install the package with this error:

## Steps to Reproduce the Problem

  1. run npm install

## Specifications

  - Datadog Lambda Layer version: N/A
  - Node version: 12.19.0
  
## Stacktrace
  
  ```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for hot-shots@true7.7.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'datadog-lambda-js'
npm ERR! notarget
  ```

**Comment by DarcyRaynerDD** [comment]
Hi @afittz, thanks for the report. We just published version 3.33.0, which should fix the issue. Can you let us know if that works for you?

**Comment by afittz** [comment]
@DarcyRaynerDD It works! Thanks for the quick fix.
